### PR TITLE
TNO-2241: Update header actions on frontpage pages

### PR DESCRIPTION
**Pull request by mihailistov**
- Add page icon to header if available for every page
Screenshot 2024-02-05 at 3 54 57 PM

- Changed content header nav back button to new mock ups
Screenshot 2024-02-05 at 3 55 06 PM

- Removed ContentActionBar action items for front page images
Screenshot 2024-02-05 at 6 07 12 PM

- Keep action items for non-front page images
Screenshot 2024-02-05 at 6 07 23 PM

### CODE REVIEW

File: app/subscriber/src/components/tool-bar/ContentActionBar.tsx [hunk 1]
```diff
@@ -33,13 +34,56 @@ export const ContentActionBar: React.FC<IContentActionBarProps> = ({
...
+  const showActionsItems = !isFrontPageImage;
+
+  React.useEffect(() => {
+    // Every time content changes, assume it's a front page image until we know it's not
+    // this prevents rendering the action items until we know they're needed
+    if (contentId !== content?.[0]?.id) {
+      setIsFrontPageImage(true);
+      setContentId(content?.[0]?.id);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (Fosol) [inline app/subscriber/src/components/tool-bar/ContentActionBar.tsx:51]: Why is this needed?  What will happen if you include `contentId`?

File: app/subscriber/src/components/tool-bar/ContentActionBar.tsx [hunk 2]
```diff
@@ -33,13 +34,56 @@ export const ContentActionBar: React.FC<IContentActionBarProps> = ({
...
+      const typeId =
+        frontPageImagesMediaTypeId ??
+        settings.find((s) => s.name === Settings.FrontPageImageMediaType)?.value;
+      // Disabling eqeqeq as settings may store number as a string or an int
+      // eslint-disable-next-line eqeqeq
+      if (typeId && typeId != content?.[0]?.mediaTypeId) {
+        setIsFrontPageImage(false);
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (Fosol) [inline app/subscriber/src/components/tool-bar/ContentActionBar.tsx:65]: Why is this needed, it looks like you have all the values already.

File: app/subscriber/src/components/tool-bar/ContentActionBar.tsx [hunk 3]
```diff
@@ -33,13 +34,56 @@ export const ContentActionBar: React.FC<IContentActionBarProps> = ({
...
+  }, [content]);
+
+  React.useEffect(() => {
+    if (isReady) {
+      const typeId =
+        frontPageImagesMediaTypeId ??
+        settings.find((s) => s.name === Settings.FrontPageImageMediaType)?.value;
+      // Disabling eqeqeq as settings may store number as a string or an int
+      // eslint-disable-next-line eqeqeq
```
Comment (Fosol) [inline app/subscriber/src/components/tool-bar/ContentActionBar.tsx:60]: You don't need to disable this, you can simply `+typeId` which will convert it to a number

File: app/subscriber/src/components/tool-bar/ContentActionBar.tsx (hunk 1 above)
Comment (mihailistov) [reply app/subscriber/src/components/tool-bar/ContentActionBar.tsx:51]: Oops, great catch. 'twas needed before but not anymore. Thanks!

**Review by Fosol — APPROVED** [review]
